.circleci/config.yml

```diff
@@ -1,17 +1,38 @@
 version: 2.1
 
 orbs:
-  php: circleci/php@1.0
   wp-svn: studiopress/wp-svn@0.1
+  php: circleci/php@1.1
+
+jobs:
+  lint:
+    docker:
+      - image: wordpress:php8.2-fpm-alpine
+    steps:
+      - checkout
+      - run:
+          name: Install Composer
+          command: |
+            mkdir -p ~/project/bin
+            cd ~/project/bin
+            echo 'export PATH=$HOME/project/bin:$PATH' >> $BASH_ENV
+            source $BASH_ENV
+            php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
+            EXPECTED_SIGNATURE=$(curl -s https://composer.github.io/installer.sig)
+            ACTUAL_SIGNATURE=$(php -r "echo hash_file('sha384', 'composer-setup.php');")
+            [[ "$EXPECTED_SIGNATURE" == "$ACTUAL_SIGNATURE" ]] && php composer-setup.php --install-dir=/root/project/bin --filename=composer || exit 1
+            rm composer-setup.php
+            composer config -g github-protocols https && composer config -g repo.packagist composer https://packagist.org
```

Comment:
We have to manually install composer because I couldn't find any pre-built PHP images that have the correct version of `curl` (8.4.0) and have Composer.